src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+    red,
+    orange,
+    yellow,
+    green,
+    lightBlue,
+    blue,
+    violet
+}
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values

Reply:
Thank you! 
I know about constants' uppercase, but i didn't comsider members of enum to be constants. I didn't want the sentence in outpoot to look odd as well, especially maven was silent about it. I'll fix it.